frontend/ui/data/creoptions.lua

```diff
@@ -136,6 +136,15 @@ local CreOptions = {
                     DCREREADER_CONFIG_DEFAULT_FONT_GAMMA,
                     DCREREADER_CONFIG_DARKER_FONT_GAMMA,
                 },
+            },
+            {
+                name = "font_hinting",
+                name_text = S.FONT_HINT,
+                toggle = {S.OFF, S.NATIVE, S.AUTO},
```

Comment:
When I saw font hinting I thought it was about the style (none, slight, medium, full). I'm less sure about the utility of this, particularly the deceptively named "auto" option. If I'm not mistaken auto overrides what the designer painstakingly put into place with a stupid algorithm.

Reply:
You're not mistaken.
Off was may be KOReader default before July 2014.
Since July 2014, Auto (Freetype algorithm) is the default (https://github.com/koreader/koreader-base/pull/199/files).
Have a look at https://github.com/koreader/koreader/issues/1375#issuecomment-186927531 for why Native/BCI (Byte-code Interpreted) is not the default. And with the few tests i did with a few fonts, Native/BCI was the worst of the 3 to me (so, Auto should probably stay the default).
But this toggle would allow everyone to check for themselves and make an opinion :)

Reply:
Also, may be the fonts I tested with didn't have good designer bytecode. May be with some other fonts, it can give a better result. Unless it's just Freetype's way of interpreted BCI that is bad, dunno.
Of interest, some posts related to the font.hinting change:
https://www.mobileread.com/forums/showthread.php?p=2882652#post2882652
https://www.mobileread.com/forums/showthread.php?p=3214470#post3214470
So, it may has a place near the Boldening/Contrast toggles.